api-reference/overview.mdx

```diff
@@ -0,0 +1,27 @@
+
+<Note>It's important to note that our API and backend SDK are intended for use exclusively in server-side applications.
```

Comment:
Add title, desctiption etc for this page

Reply:
Done.